interop/test_utils.go

```diff
@@ -673,6 +676,91 @@ func DoPickFirstUnary(tc testgrpc.TestServiceClient) {
 	}
 }
 
+func doOneSoakIteration(ctx context.Context, tc testgrpc.TestServiceClient, resetChannel bool, serverAddr string, dopts []grpc.DialOption) (latency time.Duration, err error) {
+	start := time.Now()
+	// per test spec, don't include channel shutdown in latency measurement
+	defer func() { latency = time.Since(start) }()
```

Comment:
If the purpose is to exclude time spent in conn.Close(), this needs to be deferred after `defer conn.Close()`

Deferred calls are executed in last-in-first-out order https://tour.golang.org/flowcontrol/13

Reply:
Whoops! I meant to leverage the LIFO ordering but accidentally swapped things, thanks.